ingest/Snakefile

```diff
@@ -60,7 +60,12 @@ if SEGMENTED:
         for segment in config["nucleotide_sequences"]:
             dataset_server_map[segment] = segment_identification.get(
                 "nextclade_dataset_server_map", {}
-            ).get(segment, segment_identification.get("nextclade_dataset_server"))
+            ).get(
+                segment,
+                segment_identification.get(
+                    "nextclade_dataset_server", config.get("nextclade_dataset_server")
+                ),
+            )
             dataset_name_map[segment] = segment_identification.get(
```

Comment:
Not a fan of this code here with the many nested `.get`s - seems to work but this should probably be refactored at some point into a config.py script that is not Snakemake so we can use full Pylance on it.

Reply:
lets maybe make an issue for this?

Reply:
https://github.com/loculus-project/loculus/issues/5074